reproschema/redcap2reproschema.py

```diff
@@ -252,18 +252,9 @@ def process_row(
     add_preable=True,
 ):
     """Process a row of the REDCap data and generate the jsonld file for the item."""
-    global matrix_group_count
```

Comment:
can we remove the global variable as well? i don't think this is used anywhere (please do check).

Reply:
matrix_group_count was only referenced in the chunk of code I removed, and doesn't show up anywhere else in the project. I think it is safe to remove it now.

Reply:
@Evan8456 - there is declaration at the top of the file.

Reply:
Sorry, what I meant to say was that now there is not use for matrix_group_count now so I think it is okay to remove the defalcation in addition to the chunk of code I removed earlier. I added a new commit removing the declaration.

Reply:
@satra - re `matrix_group_count` I have not removed the parts related to it, since I understood that we might be using in the near future

Reply:
this is something where we need to see an example of to see what we just removed.